src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -656,24 +663,24 @@ private static void PartialMethodChecks(SourceOrdinaryMethodSymbol definition, S
 
             PartialMethodConstraintsChecks(definition, implementation, diagnostics);
 
-            SourceMemberContainerTypeSymbol.CheckValidNullableMethodOverride(
-                implementation.DeclaringCompilation,
-                constructedDefinition,
-                implementation,
-                diagnostics,
-                (diagnostics, implementedMethod, implementingMethod, topLevel, returnTypesEqual) =>
-                {
-                    if (returnTypesEqual)
+            if (checkNullableMethodOverride)
+            {
+                SourceMemberContainerTypeSymbol.CheckValidNullableMethodOverride(
+                    implementation.DeclaringCompilation,
+                    constructedDefinition,
+                    implementation,
+                    diagnostics,
+                    (diagnostics, implementedMethod, implementingMethod, topLevel, checkNullableMethodOverride) =>
```

Comment:
Should this `checkNullableMethodOverride` parameter be renamed to something nondescript like `_` or `arg`?